matlab-src/jointSpaceLinearization/linearization1Foot_base.m

```diff
@@ -0,0 +1,194 @@
+%% linearization1Foot_base
+%  linearizes the joint space equations of motion of robot iCub when it's
+%  controlled with "stack of task" control approach. This version is for
+%  the robot balancing on one foot.
+%
+clear all
+close all
+clc
+
+%% Path and model initialization
+addpath('./../../mex-wholebodymodel/matlab/utilities');
+addpath('./../../mex-wholebodymodel/matlab/wrappers');
+addpath('./../../../../build/');
```

Comment:
This should be avoided in the code (see https://github.com/robotology/mex-wholebodymodel/issues/17). You should use a proper `pathdef.m` file or `MATLABPATH` enviromental variable. 

Feel free to ask me more details.